roles/grafana/defaults/main.yml

```
@@ -0,0 +1,9 @@
+---
+#Grafan admin variable
+user: admin
+password: strongpass
+
+#Datasource InfluxDB variable
+url: http://localhost:8086
+organization: icinga
+bucket_name: icinga
```

```suggestion
grafana_user: admin
grafana_password: strongpass

#Datasource InfluxDB variable
grafna_influxdb_url: http://localhost:8086
grafna_influxdb_organization: icinga
grafana_influxdb_bucket_name: icinga
```
Please use role name as prefix for variables to avoid collision